### DISCUSSION TITLE
Possibility of booting Android on UTM?

### DISCUSSION THREAD

**Discussion by firefinchdev** [utmapp/UTM#3332] [Ideas]
Hi
Google has confirmed that the official emulator would not make older versions of Android available via the official Android emulator on the M1 macs.
It would be really useful if UTM is able to boot Android images.
If we can boot ARM64 based images, then performance would be good enough.

Edit: typo

**Comment by conath** [comment]
This is off topic for the UTM issues section, this is for technical problems and suggestions. I will move this topic to Discussions.

I would recommend you research if it is possible to run Android in QEMU, which is the emulator used in UTM. Please let us know if you find out that it is or isn't possible.

**Reply by owoaanu** [reply]
Yes, it is possible to run Android QEMU, precisely the Androidx86 build, although I have only successfully tested version 4.4.
I am currently working on a project to make it possible for Android to run as a Virtual Machine on IOS, I am building this based off of UTM, UTM is such an inspiration👍.

**Comment by AfzalivE** [comment]
Seems like @jankais3r was able to at least boot an Android 4.4 image. Could you please share the settings? I'm not trying to boot Android 4.4, but wearOS and Android TV images.

**Comment by ty-yqs** [comment]
I am able to boot Android 4.1/4.2/4.3/4.4 image, but I have not tried 5.0+ versions. My friends said that Android 5.0+ cannot boot on UTM, but I don’t know it is true or not. I think if it is true, it might be a problem of QEMU.
